docs/explanation/parts.rst

```diff
@@ -31,6 +31,31 @@ describe a build process typically accept specifications of parts in YAML format
 mostly-declarative format. Libraries that use parts may use the underlying
 data structures to describe them.
 
+.. mermaid ::
+
+    graph LR
+
+    subgraph Part
+    CRAFT_PART_SRC
```

Comment:
I wonder if we should prefix these with $ (so `$CRAFT_PART_SRC`) to make it extra-clear that they are (also) environment variables?

Reply:
Added.